src/Filters/design.jl

```diff
@@ -241,6 +241,11 @@ function normalize_freq(w::Real, fs::Real)
     f >= 1 && error("frequencies must be less than the Nyquist frequency $(fs/2)")
     f
 end
+function normalize_complex_freq(w::Real, fs::Real)
+    f = 2*w/fs
+    f >= 2 && error("frequencies must be less than the Nyquist frequency $(fs)")
```

Comment:
Should we also put a lower bound on `f`? It seems that `0 <= f < 2` and `-1 <= f < 1` would be reasonable limits, but depending on context, one or the other may be more useful. So require `-1 <= f < 2`? Looks rather arbitrary. Or maybe we put no restriction on `f` at all? Really, I don't know.